test/unit/auth/tenant.spec.ts

```diff
@@ -431,10 +460,17 @@ describe('Tenant', () => {
       expect(tenant.emailSignInConfig).to.deep.equal(expectedEmailSignInConfig);
     });
 
+    //TODO
     it('should set readonly property multiFactorConfig', () => {
       const expectedMultiFactorConfig = new MultiFactorAuthConfig({
         state: 'ENABLED',
-        enabledProviders: ['PHONE_SMS'],
+        enabledProviders: ['PHONE_SMS', 'TOTP'],
```

Comment:
As mentioned in anoher comment, TOTP might not be allowed as an "enabledProvider". We should change unit tests accordinglty.

Also can you expand on the TODO?